src/main/java/core/basesyntax/service/reports/Report.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service.reports;
+
+public interface Report {
+    void create(String path);
```

Comment:
```suggestion
    String create(String data);
```
Report creator should get some data, and produce the report. It doesn't have a responsibility to read from file, or write to file.